data/vanderMoezel_1987/metadata.yml

```diff
@@ -0,0 +1,252 @@
+    it flowers within three or four years after a fire.
+- var_in: flowering_time
+  unit_in: .na
+  trait_name: flowering_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: .na
+  methods: Twenty-six permanently marked 20 m x 10 m plots were established, which
+    were representative of a range of topographic sites and ages-since-last-fire.
+    Each site was visited monthly from March 1981 till December 1981 and a list of
+    species in flower was compiled. As sites were not visited from January to March,
+    flowering times for those months were listed as no flowering; however, flowering
+    may have been present.
+  measurement_remarks: flowering_notes
+- var_in: post_fire_flowering
+  unit_in: .na
+  trait_name: post_fire_flowering
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: .na
+  methods: Twenty-six permanently marked 20 m x 10 m plots were established, which
+    were representative of a range of topographic sites and ages-since-last-fire.
+    Each site was visited monthly from March 1981 till December 1981 and a list of
+    species in flower was compiled. As sites were not visited from January to March,
+    flowering times for those months were listed as no flowering; however, flowering
+    may have been present.
+  measurement_remarks: flowering_notes
+substitutions:
+- trait_name: fire_response
+  find: Sprout
+  replace: resprouts
+- trait_name: fire_response
+  find: Seed
+  replace: fire_killed
+- trait_name: fire_response
+  find: Sprout and Seed
+  replace: resprouts fire_killed
+- trait_name: fire_response
+  find: Seed and Sprout
+  replace: resprouts fire_killed
+- trait_name: reproductive_maturity
+  find: ≤2
+  replace: 0--2
+- trait_name: reproductive_maturity
+  find: <2
+  replace: 0--2
+- trait_name: reproductive_maturity
+  find: ≤4
+  replace: 0--4
+- trait_name: reproductive_maturity
+  find: ≤5
+  replace: 0--5
+- trait_name: reproductive_maturity
+  find: ≤5>2
+  replace: 3--5
+- trait_name: reproductive_maturity
+  find: <4
+  replace: 0--4
+- trait_name: reproductive_maturity
+  find: ≤4>2
+  replace: 2--4
+- trait_name: reproductive_maturity
+  find: ≤6
+  replace: 0--6
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: ≤2
+  replace: 0--2
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: <2
+  replace: 0--2
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: ≤4
+  replace: 0--4
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: ≤5
+  replace: 0--5
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: ≤5>2
+  replace: 2--5
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: <4>2
+  replace: 2--4
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: <4
+  replace: 0--4
+- trait_name: reproductive_maturity_secondary_after_fire
+  find: ≤4>2
+  replace: 3--4
```

Comment:
I decided that since the values are integers we should treat `<` the same as `<=`, because <2 means "by the age of 2", not "by the age of 1" which would be the meaning of --1.